packages/eslint-plugin/docs/rules/consistent-type-imports.md

```diff
@@ -48,6 +48,12 @@ type T = import('Foo').Foo;
 const x: import('Bar') = 1;
 ```
 
+## Usage with `emitDecoratorMetadata`
+
+The `emitDecoratorMetadata` compiler option changes the code the TypeScript emits. In short - it causes TypeScript to create references to value imports when they are used in a type-only location. If you are using `emitDecoratorMetadata` then our tooling will require additional information in order for the rule to work correctly.
+
+If you are using [type-aware linting](./docs/linting/TYPED_LINTING.md), then you just need to ensure that the `tsconfig.json` you've configured for `parserOptions.project` has `emitDecoratorMetadata` turned on. Otherwise you can explicitly tell our tooling to analyze your code as if the compiler option was turned on [by setting `parserOptions.emitDecoratorMetadata` to `true`](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/parser/README.md#parseroptionsemitdecoratormetadata).
```

Comment:
@JoshuaKGoldberg / @Josh-Cena do you knows know how I can do this link to the type-aware linting docs?

I tried `../../../../docs/linting/TYPED_LINTING.md`, but docusaurus failed that build (https://app.netlify.com/sites/typescript-eslint/deploys/635e718d23d0fd00090b2e67#L152)

Am I able to do this link or do I need to hard-link it to the full URL?

Reply:
I don't know - I would personally use the hard-link full URL and then ask for help.

Reply:
The two Markdown files are rendered by different plugins, so you can't link them with file paths. Josh is right—you need to use a URL link here.